cmd/record.go

```diff
@@ -75,9 +77,18 @@ func (t *Record) GetRecordConfig(path *string, proxyPort *uint32, appCmd *string
 	if *buildDelay == 30*time.Second && confRecord.BuildDelay != 0 {
 		*buildDelay = confRecord.BuildDelay
 	}
+	*passThrough = append(*passThrough, confRecord.Stubs.Filters...)
```

Comment:
This is a redundant line as we are appending Filter or passThrough any ways in the below for loop.